documentation/resource_ard.md

```diff
@@ -0,0 +1,133 @@
+ard
+===
+
+Use the **ard** resource to manage Remote Desktop settings and preferences.
+Under the hood, an **ard** resource executes the `kickstart` command, located
+in ARDAgent.app (one of the CoreServices of macOS). It has some basic actions,
+which pertain to the simple `kickstart` subcommands. It also has the more
+complicated `:configure` action, which requires some familiarity with
+`kickstart`.
+
+[Learn more about the `kickstart` command](https://support.apple.com/en-us/HT201710).
+
+Syntax
+------
+
+A **ard** resource block declares a basic description of the command configuration
+and a set of properties depending on the actions executed. For example:
+
+```ruby
+ard 'activate and configure ard' do
+  action [:activate, :configure]
+end
+```
+
+where
+
+- `:activate` activates the ARD agent
+- `:configure` configures the agent using the `kickstart` defaut commandline arguments.
+
+The full syntax for all of the properties that are available to the **ard**
+resource is:
+
+```ruby
+ard 'description' do
+  install_package                      String
+  uninstall_options                    Array,  # defaults to ['-files', '-settings', '-prefs'] if not specified
+  restart_options                      Array, # defaults to ['-agent', '-console', '-menu'] if not specified
+  users                                Array
+  privs                                Array, # defaults to ['-all'] if not specified
+  access                               String, # defaults to '-on' if not specified
+  allow_access_for                     String, # defaults to '-allUsers' if not specified
+  computerinfo                         Array
+  clientopts                           Array
+  action                               Symbol # defaults to [:activate, :configure] if not specified
+end
+```
+
+**Note:** Not all properties are compatible with each action.
+
+Actions
+-------
+
+This resource has the following actions:
+
+`:activate`
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Activate the remote desktop agent.
```

Comment:
Let's avoid HTML codes. They reduce the plaintext readability of markdown.

Reply:
It was either this or write the README in reStructuredText. Markdown doesn't support indentation.

Reply:
That's a false dichotomy. We can also format the data differently so it reads well in plaintext and in a markdown renderer. I'd argue that we are displaying tabular data here, so a table can be appropriate as well.